mod/host.py

```diff
@@ -1152,7 +1152,10 @@ def report_current_state(self, websocket):
             for symbol, value in pluginData['outputs'].items():
                 if value is None:
                     continue
-                websocket.write_message("output_set %s %s %f" % (pluginData['instance'], symbol, value))
+                if isinstance (value, float):
+                    websocket.write_message("output_set %s %s %f" % (pluginData['instance'], symbol, value))
+                if isinstance (value, str):
```

Comment:
as I mentioned on IRC before, I don't agree with this.
note https://github.com/moddevices/mod-ui/blob/master/mod/host.py#L829
the last message of output-atom from mod-host is not saved. we should be saving it per port *and* key, not a single atom message per port.
also the current code is not saving that last value anyway, so this specific code is never reached